### DISCUSSION TITLE
Eventum Intergrated with Exchange Online

### DISCUSSION THREAD

**Discussion by sabahsawor** [eventum/eventum#1478] [General]
Hi All,

Can someone guide me through how to integrate our new office 365 email with Eventum system?
I do have Eventum v3.8.2 installed on Debian machine.

**Comment by vladsf** [comment]
There is no way for this. PHP does not support XOAuth2 yet. 
https://stackoverflow.com/questions/73370661/php-connect-mailbox-office-365-with-oauth

Start up your own imap server and forward there all emails from 365.

**Reply by glensc** [reply]
Duplicate:
- https://github.com/eventum/eventum/issues/1399

**Comment by glensc** [comment]
Also, 3.8.2 is old. upgrade!

**Comment by glensc** [comment]
you may have luck with php-imap2 replacement:
- https://github.com/javanile/php-imap2/
